tests/angular/partials.spec.js

```diff
@@ -0,0 +1,22 @@
+ngDescribe({
```

Comment:
I only added this test to demonstrate that it works, I will setup the folder structure as agreed and change tests accordingly ;)